### ISSUE TITLE
row span?

### ISSUE THREAD

**Issue by carlosrochap** [PHPOffice/PHPWord#74]
how do I create cells with rowspan more than 1?

**Comment by sajuuk78** [comment]
How can I create Table like this one?
![table](https://cloud.githubusercontent.com/assets/6146142/11626031/c4650fea-9cdf-11e5-95e8-d8af1215e37f.png)

**Comment by markoniusz** [comment]
ivanlanin - your code in my PHPWord 0.6.3 version gives me something like that. Why? 
<a href="https://abiway.pl/wymiana/table.png" >images here</a>
What is wrong? I took code directly from your example.

**Comment by markoniusz** [comment]
And one more question - I have no possibility to use Composer. Where can I found pack PHPWord code which I can use without Composer (autoload.php and Vendor)?

**Comment by gabrielbull** [comment]
@markoniusz 
You could build your own autoloader or use composer to build a "vendor" directory that you then commit on your project. I fail to see an instance where you can add code to a project but not use composer to create that code.

**Comment by kaystrobach** [comment]
Package it yourself as stated
Typically i use a build system build a bundle of all my dependencies (you can do it locally on your maschine) and upload the result

Von meinem iPhone gesendet

**Comment by danilocarva9** [comment]
Can't I use rowspan on `<td>` while using Html? It's not working, only colspan...

**Comment by SuperHaoHaoHao** [comment]
> 是啊。下面是结果的截图：
> 
> ![2014-03-10-211025-ss](https://camo.githubusercontent.com/23d81a4473a669e3d5135a07cc152d99ecfff30c/68747470733a2f2f662e636c6f75642e6769746875622e636f6d2f6173736574732f31353331332f323337343239352f63633336363438612d613835642d313165332d383134372d3862623335356363616263642e706e67)
> 
> 价值[vMerge](http://www.schemacentral.com/sc/ooxml/e-w_vMerge-1.html)不是整数。它是[一根绳子](http://www.schemacentral.com/sc/ooxml/t-w_ST_Merge.html)任一种`restart`或`continue`..我们把空白细胞`vMerge = continue`.

Invalid style value: fusion Options:restart,continue

**Comment by kaystrobach** [comment]
@SuperHaoHaoHao english please - otherwise it’s hard to follow the discussion

**Comment by madeingnecca** [comment]
I am also interested in knowing how to build a table like the one posted by @sajuuk78

**Comment by samsul97** [comment]
![image](https://user-images.githubusercontent.com/40994363/213617882-2b52c5dc-da7c-4f33-b33f-0282a74d3143.png)
Hello Im sorry, may I ask? How can I create Table like this one? Thank you.

**Comment by markonikolicdir** [comment]
> Yup. Here's the screenshot of the result:
> 
> ![2014-03-10-211025-ss](https://camo.githubusercontent.com/835422aa02071ce04a4c45d335095bfbf0001b8b9ff991ff927e85fe4a0f0d9b/68747470733a2f2f662e636c6f75642e6769746875622e636f6d2f6173736574732f31353331332f323337343239352f63633336363438612d613835642d313165332d383134372d3862623335356363616263642e706e67)
> 
> The value of [vMerge](http://www.schemacentral.com/sc/ooxml/e-w_vMerge-1.html) is not an integer. It's [a string](http://www.schemacentral.com/sc/ooxml/t-w_ST_Merge.html) of either `restart` or `continue`. We put the blank cell with `vMerge = continue`.

I

> See `Sample_09_Tables.php` on my latest commit.
> 
> ```
> $section->addTextBreak(1);
> $section->addText("Table with colspan and rowspan", $header);
> 
> $styleTable = array('borderSize' => 6, 'borderColor' => '999999');
> $cellRowSpan = array('vMerge' => 'restart', 'valign' => 'center');
> $cellRowContinue = array('vMerge' => 'continue');
> $cellColSpan = array('gridSpan' => 2, 'valign' => 'center');
> $cellHCentered = array('align' => 'center');
> $cellVCentered = array('valign' => 'center');
> 
> $PHPWord->addTableStyle('Colspan Rowspan', $styleTable);
> $table = $section->addTable('Colspan Rowspan');
> $table->addRow();
> $table->addCell(2000, $cellRowSpan)->addText('A', null, $cellHCentered);
> $table->addCell(4000, $cellColSpan)->addText('B', null, $cellHCentered);
> $table->addCell(2000, $cellRowSpan)->addText('E', null, $cellHCentered);
> $table->addRow();
> $table->addCell(null, $cellRowContinue);
> $table->addCell(2000, $cellVCentered)->addText('C', null, $cellHCentered);
> $table->addCell(2000, $cellVCentered)->addText('D', null, $cellHCentered);
> $table->addCell(null, $cellRowContinue);
> ```

When I try to create pdf from this example, in column "E" missing "rowspan", do you know what can be a problem?
